rpcs3/Emu/VFS.cpp

```diff
@@ -42,6 +42,12 @@ bool vfs::mount(std::string_view vpath, std::string_view path)
 		return false;
 	}
 
+	if (!fs::is_dir(std::string(path)) && !fs::create_dir(std::string(path)))
```

Comment:
It is a feature in some cases, directing to a non-existsting directiries that is. Do it in the syscall.

Reply:
Added a flag (default: false) to determine whether to create a new directory when there is not existing one.

Reply:
Okay.